src/component/searchBar/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const SearchBar = ({searchTerm}) => {
+  const [searchInput, setSearchInput] = React.useState("");
+
+  const handleSearch = ({ target }) => {
```

Comment:
Why duplicate state?
Better trying to work with a SSOT (single source of truth)

The same way you passed the searchTerm prop you can pass the value of the input as a proop